manifests/h/H2IK/JoystickGremlin/13.3.0/H2IK.JoystickGremlin.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: H2IK.JoystickGremlin
+PackageVersion: 13.3.0
+MinimumOSVersion: 10.0.0.0
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: msi
+  Scope: machine
+  InstallerUrl: https://github.com/WhiteMagic/JoystickGremlin/releases/download/Release_13.3/Joystick.Gremlin.R13.3.msi
+  InstallerSha256: B1A2CC9842DF3A876F3481234D0593CB764341F609A4202F7F70DF71A348759D
+#  ProductCode: 
```

Comment:
```suggestion
  ProductCode: "{0DAD4221-C8CF-4424-8DCD-3886274E89EF}"
```

Reply:
How to find ProductCode?

Reply:
In the registry, `\Microsoft\Windows\CurrentVersion\Uninstall`.

Reply:
Ok